artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis

```diff
@@ -105,6 +105,16 @@ if [ -z "$LOG_MANAGER" ] ; then
   LOG_MANAGER="$ARTEMIS_HOME/lib/${logmanager}"
 fi
 
+# determine which native library version to load
+LIBRARY_PATH="-Djava.library.path=\"$ARTEMIS_HOME/bin/lib"
+if [ "$(uname -m)" = "x86_64" ]; then
+  LIBRARY_PATH="$LIBRARY_PATH/linux-x86_64\""
+elif [ "$(uname -m)" = "i686" ]; then
+  LIBRARY_PATH="$LIBRARY_PATH/linux-i686\""
+else
+  LIBRARY_PATH=""
```

Comment:
We need something here in case of else.

Say you are on a Linux for a new System, and the user compiled the library himself. on that case you need something on the LD_LIBRARY_PATH to work.

Maybe we could have the x86_64 here?